shared/params/mainnet_config.go

```diff
@@ -75,12 +75,12 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	SlotsPerEpoch:                    32,
 	MinSeedLookahead:                 1,
 	MaxSeedLookahead:                 4,
-	EpochsPerEth1VotingPeriod:        32,
+	EpochsPerEth1VotingPeriod:        64,
 	SlotsPerHistoricalRoot:           8192,
 	MinValidatorWithdrawabilityDelay: 256,
 	ShardCommitteePeriod:             256,
 	MinEpochsToInactivityPenalty:     4,
-	Eth1FollowDistance:               1024,
+	Eth1FollowDistance:               2048,
 	SafeSlotsToUpdateJustified:       8,
 	SecondsPerETH1Block:              13,
```

Comment:
This doesn't match spec. https://github.com/ethereum/eth2.0-specs/blob/579da6d2dc734b269dbf67aa1004b54bb9449784/configs/mainnet/phase0.yaml#L48.

In reality, eth1 block times are 13 seconds on average and Prysm's eth1 connection sends too many requests this value is wrong. Thanks to @nisdas for investigating.  Discord discussion [link](https://discord.com/channels/476244492043812875/483017808658169866/776609050757431318)

Reply:
I've asked @nisdas to file an issue so we can outline our spec config deviation and potentially have the spec correct this value.

Reply:
It's also worth noting that current master has been at 13s, This `v1.0.0` branch did not make this particular change.
With that said, this was the only field that did not align with latest config

https://github.com/prysmaticlabs/prysm/blob/master/shared/params/mainnet_config.go#L85

Reply:
https://github.com/ethereum/eth2.0-specs/issues/2132